examples/sse/src/__main__.py

```diff
@@ -0,0 +1,17 @@
+import logging
```

Comment:
Because this file exists, you can add a poetry command:

```sh
poetry run sse
```

In the conf something like
```toml
[tool.poetry.scripts]
sse = 'src'
```

Reply:
For this the ideal would be to have a `src` with modules inside like `src/sse`